protonup-rs/src/download.rs

```diff
@@ -142,6 +142,10 @@ pub async fn run_quick_downloads() {
             }
         };
 
+        if files::check_if_exists(app_inst.default_install_dir(), download.output_dir(&app_inst.into_app().app_wine_version())) && !force {
```

Comment:
```suggestion
        if files::check_if_exists(app_inst.default_install_dir(), download.output_dir(&wine_version)) && !force {
```

We can reuse the wine_version Variant variable